pkg/virtctl/create/vm/vm.go

```diff
@@ -56,9 +56,17 @@ const (
 	CloudInitNetworkDataFlag   = "cloud-init-network-data"
 	InferInstancetypeFlag      = "infer-instancetype"
 	InferPreferenceFlag        = "infer-preference"
+	VolumeImportFlag           = "volume-import"
 
 	cloudInitDisk    = "cloudinitdisk"
 	inferNoOptDefVal = "inferNoOptDefVal"
+	Blank            = "blank"
```

Comment:
Can you please add support for configmaps?
Configmaps are needed for VMs which e.g. use sysprep

Reply:
I am not sure if it would be possible, because there is no configmap option in `v1beta1.DataVolumeSource` - https://kubevirt.io/cdi-api-reference/v1.56.0/definitions.html#_v1beta1_datavolumesource.

Reply:
AFAIU you are trying to make an virtctl import for http://kubevirt.io/api-reference/v0.59.0/definitions.html#_v1_volume, right?
If you scroll down, there is an attribute sysprep with definition http://kubevirt.io/api-reference/v0.59.0/definitions.html#_v1_sysprepsource.
In tekton pipelines I need this attribute to be able to use virtctl for creating VMs

Reply:
Not quiet, it's adding support for importing through `CDI` `DataVolumes` so `sysprep` isn't valid. We should add support for that elsewhere however.